test/langtools/ProblemList.txt

```diff
@@ -49,7 +49,6 @@ tools/javac/annotations/typeAnnotations/referenceinfos/Lambda.java
 tools/javac/annotations/typeAnnotations/referenceinfos/NestedTypes.java         8057687    generic-all    emit correct byte code an attributes for type annotations
 tools/javac/warnings/suppress/TypeAnnotations.java                              8057683    generic-all    improve ordering of errors with type annotations
 tools/javac/modules/SourceInSymlinkTest.java                                    8180263    windows-all    fails when run on a subst drive
-tools/javac/importscope/T8193717.java                                           8203925    generic-all    the test requires too much memory
```

Comment:
I believe that we want to do some more research regarding why the test fails for a large number of imports